tensorboard/plugins/mesh_visualizer/mesh_plugin_test.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests the Tensorboard mesh plugin."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import os
+import shutil
+from mock import patch
+import numpy as np
+import tensorflow as tf
+
+from tensorflow_graphics.tensorboard.mesh_visualizer import mesh_plugin
+from tensorflow_graphics.tensorboard.mesh_visualizer import mesh_summary
+from tensorflow_graphics.tensorboard.mesh_visualizer import plugin_data_pb2
+from tensorflow_graphics.tensorboard.mesh_visualizer import test_utils
+from werkzeug import test as werkzeug_test
+from werkzeug import wrappers
+from google3.third_party.tensorboard.backend import application
+from google3.third_party.tensorboard.backend.event_processing import plugin_event_multiplexer as event_multiplexer
+from google3.third_party.tensorboard.plugins import base_plugin
+
+
+class MeshPluginTest(tf.test.TestCase):
```

Comment:
You may need to annotate the class with run_v1_only.
https://github.com/tensorflow/tensorboard/blob/1b623e7c914558b0e521d946fccee2b8ebdb53b9/tensorboard/util/encoder_test.py#L59-L60
context: https://github.com/tensorflow/tensorboard/blob/master/tensorboard/util/test_util.py#L162-L164